src/controller.coffee

```diff
@@ -3861,13 +3872,15 @@ Editor::enablePalette = (enabled) ->
         #@paletteWrapper.style.left = '-9999px'
 
         @currentlyAnimating = false
-
-        @redrawMain()
+        @resize()
+        @resizeNubby()
+        @paletteWrapper.style.visibility = "hidden"
```

Comment:
I'm not totally sure I follow this particular change.

Reply:
(Assuming you are talking about the "visibility" change)

Good point. This was left over from an earlier attempt. Thanks for catching it. I'll remove this and the other "visibility" line.